app/renderer/js/tray.js

```diff
@@ -154,10 +154,7 @@ const createTray = function () {
 	]);
 	window.tray.setContextMenu(contextMenu);
 	window.tray.on('click', () => {
-		// Click event only works on Windows
-		if (process.platform === 'win32') {
-			ipcRenderer.send('toggle-app');
-		}
+		ipcRenderer.send('toggle-app');
```

Comment:
Is this now supported across all OS?

Reply:
I tested it on Linux. Windows was supported already. I haven't got access to MacOs, but I saw other other apps (ex. bitwarden) use the same method for all OSes